### PR TITLE
Added README

### PR DESCRIPTION
**Pull request by santhoshvaddi**
Renamed root folder name from "dialogflow" to "tokenservice"

### CODE REVIEW

File: functions/tokenservice/README.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  
+[0]: https://cloud.google.com
+[1]: https://github.com/googleapis/google-auth-library-swift/tree/master/Sources/OAuth2/FirebaseFunctionTokenProvider
+[2]: https://cloud.google.com/functions/docs/emulator
+[3]: https://github.com/GoogleCloudPlatform/ios-docs-samples.git
+[4]: https://github.com/GoogleCloudPlatform/android-docs-samples
+[5]: https://nodejs.org/en/
+[6]: https://firebase.google.com/docs/cli
+[7]: https://console.cloud.google.com
+[8]: https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/master/functions/dialogflow/functions/index.js
```
Comment (nnegrey) [inline functions/tokenservice/README.md]: dialogflow --> tokenservice
`https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/master/functions/tokenservice/functions/index.js`

File: functions/tokenservice/README.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+```
+git clone https://github.com/GoogleCloudPlatform/nodejs-docs-samples.git
+cd nodejs-docs-samples/functions/tokenService
+
+```
+  
+1. Deploy the "getOAuthToken" function with an HTTP trigger:
+
+  `functions deploy getOAuthToken --trigger--http`
```
Comment (nnegrey) [inline functions/tokenservice/README.md]: We don't use an HTTP trigger.

File: functions/tokenservice/README.md (hunk 1 above)
Comment (santhoshvaddi) [reply functions/tokenservice/README.md]: Done :-)

File: functions/tokenservice/README.md (hunk 2 above)
Comment (santhoshvaddi) [reply functions/tokenservice/README.md]: Updated to `firebase deploy --only functions`

**Pull request merged**